aspnetcore/security/key-vault-configuration/sample/Markup.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace KeyVaultConfigProviderSample
+{
+    internal static class Markup
+    {
+        internal const string Text = @"<!DOCTYPE html>

```

Comment:
I think this is fine. It's not my first choice for simple templating but I don't think we have anything better. And I think MVC might be overkill here.